container-storage-setup.sh

```diff
@@ -2501,7 +2551,10 @@ done
 # Check subcommands
 case $# in
   0)
-    CONTAINER_THINPOOL=docker-pool
+    # A lot of code references this currently; it used to be
+    # written in the config file by default as well.  Set it to the default
+    # until we can rework the code to do "check if set, otherwise use default".
+    CONTAINER_THINPOOL=${_DEFAULT_CONTAINER_THINPOOL}
```

Comment:
Current logic is that we start with default and then source /etc/sysconfig/docker-storage-setup and if user has specified a different name, that new name will take effect. So this comment is not accurate. May be its not needed at all.

Reply:
My patch no longer writes it into the default config file, but the lvm path still relies on there being a default, right?